pkg/config/config.go

```diff
@@ -185,7 +186,7 @@ func load(generalFilePath, hostsFilePath string) (*Config, error) {
 	}
 
 	if generalMap == nil || generalMap.Empty() {
```

Comment:
Is this `Empty()` check required anymore? Seems like it being empty would be equivalent to creating a new map?